src/mpr/utility/getParams.ts

```diff
@@ -0,0 +1,198 @@
+/**
+ * EC-LAB TECHNIQUE PARAMS
+ */
+type Technique = string;
+type ArraySS = [string, string][];
+export function getParams(id: number): [Technique, ArraySS] {
+  switch (id) {
+    case 0x4:
+      return ['GCPL', gcplParams];
+    case 0x6:
+      return ['CV', cvParams];
+    case 0xb:
+      throw new Error(`Not implemented technique: OCV`);
+    case 0x18:
+      return ['CA', caParams];
+    case 0x19:
+      return ['CP', cpParams];
+    case 0x1c:
+      return ['WAIT', waitParams];
+    case 0x1d:
+      throw new Error(`Not implemented technique: PEIS`);
+    case 0x1e:
+      throw new Error(`Not implemented technique: GEIS`);
+    case 0x32:
+      return ['ZIR', zirParams];
+    case 0x6c:
+      return ['LSV', lsvParams];
+    case 0x7f:
+      throw new Error(`Not implemented technique: MB`);
+    default:
+      throw new Error(
+        `Not implemented technique: unknown (0x${id.toString(16)})`,
+      );
+  }
+}
+
+export const caParams: ArraySS = [
+  ['Ei', 'Float32'],
+  ['Ei_vs', 'Uint8'],
+  ['ti', 'Float32'],
+  ['Imax', 'Float32'],
+  ['Imax_unit', 'Uint8'],
+  ['Imin', 'Float32'],
+  ['Imin_unit', 'Uint8'],
+  ['dQM', 'Float32'],
+  ['dQM_unit', 'Uint8'],
+  ['record', 'Uint8'],
+  ['dI', 'Float32'],
+  ['dI_unit', 'Uint8'],
+  ['dQ', 'Float32'],
+  ['dQ_unit', 'Uint8'],
+  ['dt', 'Float32'],
+  ['dta', 'Float32'],
+  ['E_range_min', 'Float32'],
+  ['E_range_max', 'Float32'],
+  ['I_range', 'Uint8'],
+  ['I_range_min', 'Uint8'],
+  ['I_range_max', 'Uint8'],
+  ['I_range_init', 'Uint8'],
+  ['bandwidth', 'Uint8'],
+  ['goto_Ns', 'Uint32'],
+  ['nc_cycles', 'Uint32'],
+];
+
+export const cpParams: ArraySS = [
+  ['Is', 'Float32'],
+  ['Is_unit', 'Uint8'],
+  ['Is_vs', 'Uint8'],
+  ['ts', 'Float32'],
+  ['EM', 'Float32'],
+  ['dQM', 'Float32'],
+  ['dQM_unit', 'Uint8'],
+  ['record', 'Uint8'],
+  ['dEs', 'Float32'],
+  ['dts', 'Float32'],
+  ['E_range_min', 'Float32'],
+  ['E_range_max', 'Float32'],
+  ['I_range', 'Uint8'],
+  ['bandwidth', 'Uint8'],
+  ['goto_Ns', 'Uint32'],
+  ['nc_cycles', 'Uint32'],
+];
+
+export const cvParams: ArraySS = [
+  ['Ei', 'Float32'],
+  ['Ei_vs', 'Uint8'],
+  ['dE/dt', 'Float32'],
+  ['dE/dt_unit', 'Uint8'],
+  ['E1', 'Float32'],
+  ['E1_vs', 'Uint8'],
+  ['step_percent', 'Uint8'],
+  ['N', 'Uint32'],
+  ['E_range_min', 'Float32'],
+  ['E_range_max', 'Float32'],
+  ['I_range', 'Uint8'],
+  ['I_range_min', 'Uint8'],
+  ['I_range_max', 'Uint8'],
+  ['I_range_init', 'Uint8'],
+  ['bandwidth', 'Uint8'],
+  ['E2', 'Float32'],
+  ['E2_vs', 'Uint8'],
+  ['nc_cycles', 'Uint32'],
+  ['reverse_scan', 'Uint8'],
+  ['Ef', 'Float32'],
+  ['Ef_vs', 'Uint8'],
+];
+
+export const gcplParams: ArraySS = [
```

Comment:
I don't know if this should be an array of 2 elements or directly an object:

{
'Set_I/C': 'Uint8',
...}

Or an array of object.

@santimirandarp I let you decide.

Reply:
It was turned into an array of objects (needs to guarantee the order for now.)